gitlab-examples/gitlab-pipline/gitlab-ci.yml

```diff
@@ -0,0 +1,28 @@
+image: gradle:latest
+
+before_script:
+  # this variable is used to tell gradle from which repository to download the build-info-extractor.jar, should be server-id/repo-name
+  - JFROG_CLI_EXTRACTORS_REMOTE=$ARTIFACTORY_SERVER_ID/$ARTIFCATORY_REPO
+
+  # Download JFrog CLI
+  -  curl -fL https://install-cli.jfrog.io | sh
+  # configure artifactory server
+  - jf config add $ARTIFACTORY_SERVER_ID --url=$ARTIFACTORY_URL --user=$ARTIFACTORY_USER --password=$ARTIFACTORY_PASS
+  - jf config show
+
+  # configure gradle, set upload and download repos
+  - jf gradlec --repo-resolve $ARTIFCATORY_REPO --repo-deploy $ARTIFCATORY_REPO
+
+gitlab-ci-gradle-build: #This job name will be used as the build info repository name in artifactory
+  stage: build
+  script:
+    # upload the built artifact to artifactory
+    - jf gradle clean artifactoryPublish -b build.gradle --build-name=$CI_JOB_NAME --build-number=$CI_JOB_ID
```

Comment:
```suggestion
    - jf gradle clean artifactoryPublish --build-name=$CI_JOB_NAME --build-number=$CI_JOB_ID
```

Reply:
Did not work without suppling the path to build.gradle

Reply:
Removed the -b flag after testing it works